src/common/config_opts.h

```diff
@@ -1390,6 +1390,7 @@ OPTION(rbd_mirroring_replay_delay, OPT_INT, 0) // time-delay in seconds for rbd-
  * rbd_create3()/RBD::create3() and rbd_clone2/RBD::clone2() are only
  * affected by rbd_default_order.
  */
+OPTION(rbd_default_pool, OPT_STR, "rbd") // default pool for storing images
```

Comment:
Just an idea and might be done as a separate PR. We could add a validator for this option. This would make error messages have more sense when e.g. rbd_default_pool is set to empty.